lib/bowling_cases.rb

```diff
@@ -0,0 +1,47 @@
+class BowlingCase < OpenStruct
+  def test_name
+    'test_%s' % description.downcase.tr(' ', '_')
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  def work_load
+    if assert_error?
```

Comment:
You're checking `assert_error?` twice, (again on line 30)
What if you moved all this logic into the `assert` method?